### ISSUE TITLE
Potential Typo in Loss Objective

### ISSUE THREAD

**Issue by ThisGame42** [JiachengCheng96/NDCC#1]
Hi @JiachengCheng96 , just wanted to say great work and I really enjoyed reading your paper. It was probably the most enjoyable paper I've read in 2024! 

Anyway, I was going through the codebase and noticed something that could potentially be a typo?  In [line 101 of NDCC.py](https://github.com/JiachengCheng96/Learning-Deep-Classifiers-Consistent-with-Fine-Grained-Novelty-Detection/blob/main/NDCC.py#L101), the loss objective is defined as 
```python
loss = loss_CE + self.lmd * (loss_MD + self.gma * loss_MD)
```
which seems incorrect as the loss objective is defined as 
```python
loss = loss_CE + self.lmd * (loss_MD + self.gma * loss_NLL)
```
in your paper. Could you please clarify if my understanding is correct? Thanks a lot in advance.
